Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2635,6 +2635,20 @@ EnableInheritURIQueryComponent:
     WebCore:
       default: false
 
+EnableSVGDeprecatedProperties:
+  type: bool
+  status: preview
```

Comment:
I think the preference is doing the opposite of what you want here.

By default, the deprecated properties would be disabled, and in STP the deprecated properties would be enabled.

@brentfulgham can you confirm this is how it would work?

Reply:
This is my second attempt to add preferences, so I am expecting to mess-up things but it is also learning opportunity. :-)

Reply:
We should use `status: stable` here. Otherwise, STP will turn this feature on for anyone launching the app.

I assume the goal is to have the deprecated properties off by default when you use Safari (just like they are in Firefox and Chrome). Using `stable` will put the flag in the right place, but allow the user to turn the flag back on if they need it.